sormas-ui/src/main/java/de/symeda/sormas/ui/samples/environmentsample/EnvironmentSampleEditForm.java

```diff
@@ -107,7 +104,7 @@ public class EnvironmentSampleEditForm extends AbstractEditForm<EnvironmentSampl
 		EnvironmentSampleDto.LAB_SAMPLE_ID,
 		EnvironmentSampleDto.SPECIMEN_CONDITION);
 
-	private CheckBoxTree<WeatherCondition> weatherConditionCheckBoxTree;
+	private CheckBoxTreeUpdated<WeatherCondition> weatherConditionCheckBoxTreeUpdated;
```

Comment:
can be converted to local variable

Reply:
cannot be converted since it is used in a different method on Disabling and Enabling fields